tests/conftest.py

```diff
@@ -15,6 +15,7 @@
 
 # Need to disable early here before any feast imports (too late to patch this in an autouse fixture)
 os.environ["FEAST_USAGE"] = "False"
+os.environ["MODE"] = "development"
```

Comment:
Would it make sense to make the environment variable name more specific?

Reply:
Any name suggestion in your mind? Thinking of using it to control all the cases where we want to raise the exception during development but not the production.

Reply:
How about `FEATUREBYTE_EXEC_MODE`?